### PR TITLE
Rfblock tableflip

### PR DESCRIPTION
**Pull request by rfblock**
please accept luke i'll give one free blowie coupon from jp

### CODE REVIEW

File: src/main/java/com/hackclub/hccore/commands/TableflipCommand.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    @Override
+    public boolean onCommand(CommandSender sender, Command cmd, String alias, String[] args) {
+        if (!(sender instanceof Player)) {
+            sender.sendMessage(ChatColor.RED + "You must be a player to use this");
+            return true;
+        }
+
+        Player player = (Player) sender;
+        if (args.length == 0) {
+            player.chat(TableflipCommand.SHRUG);
```
Comment (lukec11) [inline src/main/java/com/hackclub/hccore/commands/TableflipCommand.java]: You've missed a step on your copy paste here!

File: src/main/java/com/hackclub/hccore/commands/TableflipCommand.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        } else {
+            player.chat(String.join(" ", args) + " " + TableflipCommand.SHRUG);
```
Comment (lukec11) [inline src/main/java/com/hackclub/hccore/commands/TableflipCommand.java]: You've missed another one!

File: src/main/java/com/hackclub/hccore/commands/TableflipCommand.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+import com.hackclub.hccore.HCCorePlugin;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+public class TableflipCommand implements CommandExecutor {
+    private static final String SHRUG = "(╯°□°）╯︵ ┻━┻";
```
Comment (lukec11) [inline src/main/java/com/hackclub/hccore/commands/TableflipCommand.java]: why is it named `SHRUG`?

File: src/main/java/com/hackclub/hccore/HCCorePlugin.java [hunk 4]
```diff
@@ -57,6 +58,7 @@ public void onEnable() {
         this.getCommand("shrug").setExecutor(new ShrugCommand(this));
         this.getCommand("spawn").setExecutor(new SpawnCommand(this));
         this.getCommand("stats").setExecutor(new StatsCommand(this));
+        this.getCommand("tableflip").setExecutor(new ShrugCommand(this));
```
Comment (lukec11) [inline src/main/java/com/hackclub/hccore/HCCorePlugin.java]: if you are going to copy paste, at least do it right.

**Comment by lukec11** [comment]
@rfblock please fix the issues above!

**Comment by ifvictr** [comment]
@rfblock You need to add the command to `plugin.yml`.

**Comment by lukec11** [comment]
:tada:

**Pull request merged**